tests/snapshots.rs

```diff
@@ -297,6 +297,10 @@ fn convert_wgsl() {
             "globals",
             Targets::SPIRV | Targets::METAL | Targets::GLSL | Targets::WGSL,
         ),
+        (
+            "baked-load",
```

Comment:
is a new test really necessary here? I'd think that this code already affects all the snapshot tests. I don't think it needs to be tested exclusively.

Reply:
This test case is based on #910. If we didn't catch this error previously, we need this case, to be sure we don't break it again.

Reply:
I see the change here in this PR as pure formatting. There is no semantical changes expected to be here.
So adding a test case for #910 is not something that should be a part of this.

Reply:
Ok, let's look at the wgsl code:
```wgsl
var a: f32 = 0.0;
// We should check that backed doesn't skip this expression
let b: f32 = a;
a = 1.0;
v_data.data = b;
```

Previously we are skipped `let b: f32 = a;` because it's has not enough ref count to be a baked expression. But now can track every `let` expressions and generate correct code without any additional unnecessary code. (like we tried in #910 and #925).

Reply:
Ok, thank you for explaining. I think there is a bit of confusion about what this PR does.

I want it to be in line with all the other names we have in IR: all of them are non-semantical and optional. This matches SPIR-V. In this sense, the PR itself is non-semantical.

I.e. adding a name to an expression doesn't change the meaning of the program (in terms of IR semantics). This PR isn't here to fix the issue. The issue needs to be fixed by adjusting the bake ref count, instead - #925.

Reply:
Agree with you, thank you! I will remove this part from PR and will update #925. But we definitely need to figure out how to reduce the amount unnecessary code in such case.